### DISCUSSION TITLE
红线：关于风雨阁Storm Pavilion的那些事（一） - 夏沫花火zzz🌙 (Muska_Ami)のLife

### DISCUSSION THREAD

**Discussion by Muska-Ami** [Muska-Ami/muska-ami-blog#56] [General]
https://blog.1l1.icu/2023/10/20/hong-xian-guan-yu-feng-yu-ge-storm-pavilion-de-na-xie-shi-yi/ 

 上次提起风雨阁已过去多时，在后面这段时间发生了挺多事情，所以又将旧事重新提起，做一些补记。由于要补充的东西比较多，我会分几篇博文发布。 起源：来自感性的联系关于风雨阁的前世今生，在上一次有提到过。 风雨阁Storm Pavilion源于它的前身——霜雪樱花Frost Snow Cherry Blossoms。这里是这场梦开始的地方。后来，风雨阁稀里糊涂的出世了。现在风雨阁的创始日期已不可考，我也不

**Comment by SRY-CTB** [comment]
大佬带我入坑，被HTML吸引到了

**Comment by Muska-Ami** [comment]
> @SRY-CTB
> 大佬带我入坑，被HTML吸引到了
> 

捉
